sql/hive-thriftserver/v1.2/src/main/java/org/apache/hive/service/cli/operation/OperationManager.java

```diff
@@ -87,7 +87,7 @@ private void initOperationLogCapture(String loggingMode) {
   }
 
   public ExecuteStatementOperation newExecuteStatementOperation(HiveSession parentSession,
-      String statement, Map<String, String> confOverlay, boolean runAsync)
+      String statement, Map<String, String> confOverlay, boolean runAsync, long queryTimeout)
```

Comment:
Should we change these at all since `queryTimeout` is supported only in hive-2.3?

Reply:
ditto

Reply:
This update is needed to add the `queryTimeout` param in `SparkSQLOperationManager#newExecuteStatementOperation`: 
https://github.com/apache/spark/pull/28991/files#diff-9d2cd65aaeae992250b5f40d8c289287R48
Looked around the related code and I think the current one is the simplest. If you have better idea or another suggestion, please let me know in #29933.